### PR TITLE
add logbook

### PR DESCRIPTION
**Pull request by msarahan**
Taken originally from https://github.com/quantopian/zipline/tree/master/conda/logbook

Pinging @mitsuhiko @gbrandl @scilifelab @brainstorm @vmalloc @jikamens @richafrank

Because of your previous maintenance of this package or the original recipe, I have listed you as maintainer for this package on conda-forge, a library of community-build conda packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/logbook-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that is entirely fine too - I'm happy to remove you from the list below.

### CODE REVIEW

**Comment by jakirkham** [comment]
Think there was a typo with @gbrandl. Not sure who was intended.

**Comment by msarahan** [comment]
I copied the maintainers from the PyPI site: https://pypi.python.org/pypi/Logbook - he/she either has a different github name, or no account.

**Comment by jakirkham** [comment]
I think they changed it to @birkenfeld.

File: recipes/logbook/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name="logbook" %}
+{% set name_upper=name|title %}
```
Comment (jakirkham) [inline recipes/logbook/meta.yaml]: Neat! Didn't know this little trick.

**Comment by jakirkham** [comment]
LGTM @msarahan. We should get permission before merging. Alternatively, if you would be willing to drop the non-confirmed maintainers, I would be happy to merge now.

**Comment by richafrank** [comment]
Speaking as a recipe maintainer, it looks good to me, but I'm not the package maintainer.

File: recipes/logbook/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - logbook
+
+about:
+  home: http://logbook.pocoo.org/
+  license: BSD 3-clause
```
Comment (jakirkham) [inline recipes/logbook/meta.yaml]: Including the license file would be nice. 😉

**Comment by synapticarbors** [comment]
@msarahan -- any update on the status of this? It looks like the newest version is now `1.1.0`. If you are interested in cleaning up this PR:

- rebasing up to the latest `staged-recipes`
- Updating URLS (it looks like it moved to https://github.com/getlogbook/logbook)
- add the license file
- add `dev_url` and `doc_url`

I should be able to quickly review and merge this.

**Pull request closed without merging**